### PR TITLE
PR review for error messages

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

**Review by skorati — COMMENTED** [review]
@sameshai Couple if comments on PR: https://github.com/IBM/ibmcloud-volume-file-vpc/pull/15/files

1. Make the errorCodes homogeneous . Please use if `SubnetFine*` ,  `SubnetList*` as we are using `Volume*Action*` in other places.
2. For `ListSubnetsFailed` and `ListSecurityGroupsFailed`, why are we returning 500, is this the only applicable error code.. is it possible that users will not have access or its forbidden action, can you confirm.
3. Could you rephrase the below errors, they are bit ambiguous to me.
- ` Description: "A subnet with the specified zone '%s' and available cluster subnet list '%s' could not be found.",` -> instead of `and available cluster` , can we have `in  cluster subnet list`

-  `"A securityGroup with the specified cluster securityGroup name '%s' could not be found.",` -> Its confusing if %s represents cluster name or security group name


4. The names for these errorCodes are too long can we shorten/simplify them.
- SecurityGroupFindFailedWithVPCAndSecurityGroupName
- SubnetFindFailedWithZoneAndSubnetID

We can have webex if needed.

**Comment by sameshai** [comment]
> @sameshai Couple if comments on PR: https://github.com/IBM/ibmcloud-volume-file-vpc/pull/15/files
> 
> 1. Make the errorCodes homogeneous . Please use if `SubnetFine*` ,  `SubnetList*` as we are using `Volume*Action*` in other places.
> 2. For `ListSubnetsFailed` and `ListSecurityGroupsFailed`, why are we returning 500, is this the only applicable error code.. is it possible that users will not have access or its forbidden action, can you confirm.
> 3. Could you rephrase the below errors, they are bit ambiguous to me.
> 
> * ` Description: "A subnet with the specified zone '%s' and available cluster subnet list '%s' could not be found.",` -> instead of `and available cluster` , can we have `in  cluster subnet list`
> * `"A securityGroup with the specified cluster securityGroup name '%s' could not be found.",` -> Its confusing if %s represents cluster name or security group name
> 
> 4. The names for these errorCodes are too long can we shorten/simplify them.
> 
> * SecurityGroupFindFailedWithVPCAndSecurityGroupName
> * SubnetFindFailedWithZoneAndSubnetID
> 
> We can have webex if needed.

**Comment by sameshai** [comment]
1. Make the errorCodes homogeneous . Please use if `SubnetFine*` ,  `SubnetList*` as we are using `Volume*Action*` in other places.
`Done
`
2. For `ListSubnetsFailed` and `ListSecurityGroupsFailed`, why are we returning 500, is this the only applicable error code.. is it possible that users will not have access or its forbidden action, can you confirm.

`There is no API specific access errors if we have reached till this stage that means forbidden or permission issue is not ablicable. It will fail with actual error code by IAAS back end error. This is rare case like network issue in which we will say 500 error only
`

 3. Could you rephrase the below errors, they are bit ambiguous to me.
* ` Description: "A subnet with the specified zone '%s' and available cluster subnet list '%s' could not be found.",` -> instead of `and available cluster` , can we have `in  cluster subnet list`* 
 * `"A securityGroup with the specified cluster securityGroup name '%s' could not be found.",` -> Its confusing if %s represents cluster name or security group name

`Done
`
4. The names for these errorCodes are too long can we shorten/simplify them.
SecurityGroupFindFailedWithVPCAndSecurityGroupName
SubnetFindFailedWithZoneAndSubnetID

`Done`

@skorati uploaded the changes as part of https://github.com/IBM/ibmcloud-volume-file-vpc/pull/25/files
